File: backend/src/main/java/space/davids_digital/vk_gpt_bot/model/ChatSettingsModel.java
package space.davids_digital.vk_gpt_bot.model;

public record ChatSettingsModel(
        long peerId,
        String titleCached,
        String context,
        int gptMaxInputTokens,
        int gptMaxOutputTokens,
        double gptTemperature,
        double gptTopP,
        double gptFrequencyPenalty,
        double gptPresencePenalty,
        boolean botEnabled,
        String gptModel
) {}

File: backend/src/main/java/space/davids_digital/sweetie/model/ChatSettingsModel.kt
package space.davids_digital.sweetie.model

data class ChatSettingsModel(
    val peerId: Long,
    val titleCached: String?,
    val context: String,
    val gptMaxInputTokens: Int,
    val gptMaxOutputTokens: Int,
    val gptTemperature: Double,
    val gptTopP: Double,
    val gptFrequencyPenalty: Double,
    val gptPresencePenalty: Double,
    val botEnabled: Boolean,
    val gptModel: String,
    val processAudioMessages: Boolean,
    val ttsVoice: String,
    val ttsSpeed: Double,
)
